src/Integration.Vsix.UnitTests/SonarAnalyzer/SonarAnalyzerManagerForBoundTests.cs

```diff
@@ -18,12 +18,12 @@
 namespace SonarLint.VisualStudio.Integration.UnitTests.SonarAnalyzer
 {
     [TestClass]
-    public class SonarAnalyzerDeactivationManagerForBoundTests
+    public class SonarAnalyzerManagerForBoundTests
```

Comment:
Any way just make sure to name the test files using the same prefix <TypeName>...